### PR TITLE
[SECURITY-1097] Merge 1.262.1 back into master

### PR DESCRIPTION
**Pull request by daniel-beck**
Needs merging with merge commit, DO NOT use squash-merge or rebase-merge.

### CODE REVIEW

**Review by jvz — APPROVED** [review]

File: core/src/main/java/org/kohsuke/stapler/RequestImpl.java [hunk 1]
```diff
@@ -120,6 +121,16 @@
 
     private BindInterceptor bindInterceptor = BindInterceptor.NOOP;
 
+    /**
+     * List of HTTP verbs of requests for which {@link #getSubmittedForm()} can legitimately be called.
+     * If that method is invoked during a request sent with a different verb, an exception will be thrown.
+     */
+    private static /* nonfinal for Jenkins script console */ List<String> ALLOWED_HTTP_VERBS_FOR_FORMS;
```
Comment (jglick) [inline core/src/main/java/org/kohsuke/stapler/RequestImpl.java:128]: BTW a `Set` would make more sense, not that it really matters here.

**Review by jglick — APPROVED** [review]

**Comment by jglick** [comment]
@daniel-beck can I cut a release now? You and @basil have a bunch of unreleased stuff.

**Comment by daniel-beck** [comment]
@jglick Works for me. Thanks!

**Comment by jglick** [comment]
https://github.com/stapler/stapler/releases/tag/stapler-parent-1.263

**Pull request merged**